lib/api/bucketPutObjectLock.js

```diff
@@ -1,13 +1,13 @@
 const { waterfall } = require('async');
 const arsenal = require('arsenal');
 
-const errors = arsenal.errors;
-const ObjectLockConfiguration = arsenal.models.ObjectLockConfiguration;
+const { errors } = arsenal;
+const { ObjectLockConfiguration } = arsenal.models;
```

Comment:
We should either use `.` notation or fully commit to the short hand
```suggestion
const { models: { ObjectLockConfiguration } } = arsenal;
```

Reply:
done here : https://github.com/scality/cloudserver/pull/5456/files#diff-fff39f585107c37daa87b33d580d3590a42b802201d620ae069697a5dc042893R5